src/utils/ui/i18n/cs_CZ.json

```diff
@@ -107,7 +108,9 @@
       "splitter": "Vzor rozdělení jména interpreta",
       "splitter_tooltip": "Regex pro rozdělení jmen umělců. Ponechte prázdné pro vypnutí rozdělení.",
       "scan_interval": "Scan interval",
-      "scan_interval_tooltip": "Interval in seconds to trigger a scan task. Set to -1 to disable automatic scanning"
+      "scan_interval_tooltip": "Interval in seconds to trigger a scan task. Set to -1 to disable automatic scanning",
+      "scan_threads": "Scan CPU threads",
```

Comment:
Not sure what is the context of this yet, literal translation would be "Skenovat CPU vlákna"